lib/open_api_spex/controller.ex

```diff
@@ -205,13 +205,12 @@ defmodule OpenApiSpex.Controller do
 
         {:ok, {mod_meta, summary, description, meta}}
 
-      {_, _, _, :none, %{responses: _responses} = meta} ->
-        summary = ""
-        description = ""
-        {:ok, {mod_meta, summary, description, meta}}
-
-      {_, _, _, :none, %{}} ->
-        IO.warn("No docs found for function #{module}.#{name}/2")
+      {_, _, _, :none, meta} when is_map(meta) ->
+        if Enum.empty?(meta) do
```

Comment:
https://swagger.io/specification/#operation-object

responses | Responses Object | REQUIRED. The list of possible responses as they are returned from executing this operation.

The responses are required, so we should warn if we have non-empty meta that is missing `%{responses}`

Reply:
Sorry, I will close this PR.